Source/Pawnmorphs/Esoteria/Hediffs/Comp_MutationSeverityAdjust.cs

```diff
@@ -295,7 +284,10 @@ public override string CompDebugString()
             stringBuilder.Append(base.CompDebugString());
             if (!Pawn.Dead)
             {
-                stringBuilder.AppendLine("severity/day: " + SeverityChangePerDay().ToString("F3"));
+                stringBuilder.Append($"severity/day: {SeverityChangePerDay().ToString("F3")}");
+
+                if (_halted)
+                    stringBuilder.Append(" (halted)");
```

Comment:
One last thing I'd like added to this string would be EffectiveMax, so that we can tell for sure if it's being updated properly or not

Reply:
Good point